.gitignore

```diff
@@ -60,3 +60,6 @@ build
 # Kconfig files
 sdkconfig
 sdkconfig.old
+
+# pre-commit configuration file
+.pre-commit-config.yaml
```

Comment:
I think it's fine to add it and we can even promote it's usage.

Reply:
I will upload the configuration file then 😄